From e248cbf54c7c87d235d10214fbc98772edf564f7 Mon Sep 17 00:00:00 2001
From: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
Date: Mon, 13 Nov 2017 11:40:56 +0900
Subject: [PATCH 06/23] Revert "arm64: dts: r8a7795: Add MFIS Lock device node"

This reverts commit edf4b5915805 ("arm64: dts: r8a7795: Add MFIS
Lock device node")

Signed-off-by: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a7795.dtsi | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795.dtsi b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
index 83cba567..3027c49 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
@@ -872,13 +872,6 @@
 			       <&ipmmu_mp0 30>, <&ipmmu_mp0 31>;
 		};
 
-		mfis: mfis@e6260000 {
-			compatible = "renesas,mfis-lock-r8a7795",
-				     "renesas,mfis-lock";
-			reg = <0 0xe6260000 0 0x1000>;
-			#hwlock-cells = <1>;
-		};
-
 		pfc: pfc@e6060000 {
 			compatible = "renesas,pfc-r8a7795";
 			reg = <0 0xe6060000 0 0x50c>;
-- 
1.9.1

